lib/sprinkler_web/mqtt/subscriber.ex

```diff
@@ -0,0 +1,27 @@
+defmodule SprinklerWeb.Mqtt.Subscriber do
+  @moduledoc false
+
+  use Supervisor
+
+  @broker_host Application.get_env(:sprinkler, SprinklerWeb.Endpoint)[:broker_host]
```

Comment:
This should be enough after the other suggested change

```suggestion
  @broker_host Application.get_env(:sprinkler, :broker_host)
```